docs/docs/mdx/programmatically-creating-pages.md

```diff
@@ -274,13 +274,17 @@ component should look like:
 ```jsx:title=src/components/posts-page-layout.js
 import React from "react"
 import { graphql } from "gatsby"
+import { MDXProvider } from "@mdx-js/react"
 import { MDXRenderer } from "gatsby-plugin-mdx"
+import { Link } from "gatsby"
 
```

Comment:
did you forget here also the link for the cortcodes?
```jsx
const shortcodes = { Link } // Provide common components here
```

Reply:
created PR:

- #23157 `fix(docs): update programmatically-creating-pages.md add missing shortcodes`